site/content/docs/callouts.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Callouts
+---
+
+With Flowershow you can make use of callouts when using a blockquote. This feature behaves similarly to how Obsidian handles them.
+
+**Example:**
+
+```md
+> [!info] This is cool!
+> Here's a callout block.
+> It supports **markdown** and [[Internal link|wikilinks]].
+```
+
+**Renders as:**
+
+> [!info] This is cool!
+> Here's a callout block.
+> It supports **markdown** and [[docs/index|wikilinks]].
+
+**Supported Types:**
+
+Flowershow supports 13 different Obsidian callout types (with aliases) like note, abstract, todo, or tip. See this [Obsidian docs page](https://help.obsidian.md/How+to/Use+callouts) to learn more about different callout types.
+
+- note
+- tip  (alias: hint, important)
+- warning (alias: caution, attention)
+- abstract (alias: summary, tldr)
+- info
+- todo
+- success (alias: check, done)
+- question (alias: help, faq)
+- failure (alias: fail, missing)
+- danger (alias: error)
+- bug
+- example
+- quote (alias: cite)
+
+<div className="border-2 border-slate-400 rounded-md px-4 my-4">
```

Comment:
Couldn't this be an info callout now 🙂 (let's avoid custom html wherever possible as it goes out of sync fast)